### PR TITLE
Create Energetics Meets Generex.md

### PR DESCRIPTION
**Pull request by sarikasu**

### CODE REVIEW

File: tasks/Energetics Meets Generex.md [hunk 1]
```diff
@@ -15,23 +15,23 @@ Negotiation Instructions: https://new.negotiationexercises.com/exercises/?query=
...
+3. At the end of the negotiation, each dyad completed an agreement sheet asking which party made the first offer and the value of the final selling price.
 
 
 ## Roles 
 > The seller, Energetics, represents a large energy company interested in selling its wind division. The seller’s BATNA was to sell their assets piecemeal for a profit of $170 million. 
 > The buyer, Generex, represents a large power company interested in buying the division. Buyers had two alternatives, one relatively cheap and one relatively expensive: buy a much smaller wind division in a less-attractive market for $220 million, or develop a comparable facility from scratch for $350 million with a 3-year delay.
 
 ## Instructions
-ASKED FOR MORE DETAILS 6/7
+ASKED FOR MORE DETAILS 6/7.
```
Comment (KaranSampath) [inline tasks/Energetics Meets Generex.md:26]: Have we received any response?

**Review by KaranSampath — CHANGES_REQUESTED** [review]
We'll need to check if we can find any more in-depth instructions.

File: tasks/Energetics Meets Generex.md (hunk 1 above)
Comment (KaranSampath) [reply tasks/Energetics Meets Generex.md:26]: We'll also need to add details of the alternate studies.

**Review by sarikasu — COMMENTED** [review]
No instructions from researchers- they said to reference those on the negotiations website!

**Review by KaranSampath — APPROVED** [review]

**Pull request merged**